novice-goals.Rmd

```diff
@@ -0,0 +1,207 @@
+# Outline {#outline}
+
+This outline describes the questions that the novice courses on R and Python will answer.
+The advanced course can then assume that learners have hands-on experience with these topics
+but nothing more.
+
```

Comment:
The first two look great to me - what questions do you have students investigate with the Louisville data?

Reply:
1. How do salaries change from year to year?
2. Is the a relationship amongst the police officers' salaries (RegularRate, OvertimeRate, IncentiveAllowance, Other)

We didn't get to track the salary of a single employee across years, but I think it would be cool to incorporate it, if possible.

Reply:
:+1: